pkgs/by-name/ni/nixos-firewall-tool/nixos-firewall-tool.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
```

Comment:
```suggestion
```

I think these are redundant with writeShellApplication but it doesn't hurt I guess. I'll leave this open for a day or so so that others can chime in but I think this is gtg.

Perhaps we should even have this in the path by default when the nixos-firewall is enabled.

Reply:
I'm okay with that.

I have this always in my scripts because it enables easier development. I can just do `nix develop ...` and then run the script without building it every time.